.changesets/fix_bnjjj_fix_error_log_subs.md

```diff
@@ -0,0 +1,5 @@
+### fix(subscription): do not log an error when the websocket stream has been interrupted, keep it in trace level to avoid useless noises ([PR #8344](https://github.com/apollographql/router/pull/8344))
+
+Convert log from `error` level to `trace` when the websocket stream has been interrupted to avoid useless noises.
```

Comment:
What does this mean from an end user perspective? When is a websocket stream "interrupted"?

Reply:
For example if the subgraph crashed and the connection suddenly closed. For the end user perspective it's still the same behavior, it returns an error with the error message indicating the connection has been interrupted